### PR TITLE
CI: added Druid Operator Tests

### PR DESCRIPTION
**Pull request by backstreetkiwi**
## Description
Execution of the Druid integration test in our CI.
The test has successfully run, see https://ci.stackable.tech/view/02%20Component%20Tests%20(flex)/job/Druid%20Operator%20Integration%20Tests%20(flex)/4/

## Review Checklist
- [x] Code contains useful comments
- [x] Changelog updated (or not applicable)

### CODE REVIEW

File: druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      version: 3.5.8
+      servers:
+        roleGroups:
+          default:
+            selector:
+              matchLabels:
+                kubernetes.io/os: linux
+            config:
+              adminPort: 12000
+              metricsPort: 9505
```
Comment (siegfriedweber) [inline druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml:38]: The tests will create the necessary services as soon as #40 is merged. So the ZookeeperCluster definition can be removed.

File: druid-operator/.ci/integration-tests/hcloud-centos-8/cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      version: 3.5.8
+      servers:
+        roleGroups:
+          default:
+            selector:
+              matchLabels:
+                kubernetes.io/os: linux
+            config:
+              adminPort: 12000
+              metricsPort: 9505
```
Comment (siegfriedweber) [inline druid-operator/.ci/integration-tests/hcloud-centos-8/cluster.yaml:38]: The tests will create the necessary services as soon as #40 is merged. So the ZookeeperCluster definition can be removed.

File: druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+publicKeys: []
+spec:
+  region: eu-central-1
+  wireguard: false
+  versions:
+    druid-operator: "$DRUID_OPERATOR_VERSION"
+    zookeeper-operator: NIGHTLY
+  nodes:
+    main:
+      numberOfNodes: 3
```
Comment (siegfriedweber) [inline druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml:18]: When the Druid integration tests are extended with ingestion jobs then one of the Kubernetes nodes needs the label `nodeType: druid-data`. The kind cluster setup script adds this label to one of the nodes so this should also be done on T2.

**Review by siegfriedweber — CHANGES_REQUESTED** [review]

File: druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml (hunk 1 above)
Comment (backstreetkiwi) [reply druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml:38]: Okay, I wait for #40.

File: druid-operator/.ci/integration-tests/hcloud-centos-8/cluster.yaml (hunk 2 above)
Comment (backstreetkiwi) [reply druid-operator/.ci/integration-tests/hcloud-centos-8/cluster.yaml:38]: OK.

File: druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml (hunk 3 above)
Comment (backstreetkiwi) [reply druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml:18]: I'll have a look at that. I think I'll do it early in the test script using a `kubectl` command.

File: druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml (hunk 1 above)
Comment (backstreetkiwi) [reply druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml]: ZK service definition is removed, test still runs, see: https://ci.stackable.tech/view/02%20Component%20Tests%20(flex)/job/Druid%20Operator%20Integration%20Tests%20(flex)/4/

File: druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml (hunk 3 above)
Comment (backstreetkiwi) [reply druid-operator/.ci/integration-tests/aws-centos-8/cluster.yaml:18]: Test script labels the nodes according to the kind cluster setup

**Review by siegfriedweber — APPROVED** [review]
LGTM

**Pull request merged**